cmd/storj-sim/network.go

```diff
@@ -314,6 +315,21 @@ func newNetwork(flags *Flags) (*Processes, error) {
 		process.ExecBefore["run"] = func(process *Process) error {
 			return readConfigString(&process.Address, process.Directory, "server.address")
 		}
+
+		migrationProcess := processes.New(Info{
+			Name:       fmt.Sprintf("satellite-migration/%d", i),
+			Executable: "satellite",
+			Directory:  filepath.Join(processes.Directory, "satellite", fmt.Sprint(i)),
+		})
+
+		migrationProcess.Arguments = withCommon(process.Directory, Arguments{
+			"run": {
+				"migration",
+				"--debug.addr", net.JoinHostPort(host, port(satellitePeer, i, debugMigrationHTTP)),
+			},
+		})
+
+		process.WaitForExited(migrationProcess)
```

Comment:
Add the dependency for satellite-peer as well.

Reply:
Also, this thing that you did with joining the loop... you can do it with satellite-peer.